mindmeld/models/model.py

```diff
@@ -1185,27 +1189,35 @@ def initialize_resources(self, resource_loader, examples=None, labels=None):
 
         # get list of resources required by feature extractors
         required_resources = self.config.required_resources()
-
         enable_stemming = ENABLE_STEMMING in required_resources
-
-        # load required resources if not present in model resources
+        resource_builders = {}
         for rname in required_resources:
-            if rname == ENABLE_STEMMING:
+            if rname in self._resources:
```

Comment:
I think the intention of this line is not the override a key in `self._resources`, but judging by the current code, we do override the key even if it exists in the map. Maybe we should remove this check?

Reply:
There was a check before as well.  
https://github.com/cisco/mindmeld/blob/master/mindmeld/models/model.py#L1197
This is slightly different, but it should end up equal from what I saw.

Reply:
This check is actually required to load models.  If we try to reload the resources when the models are loaded from cache it fails because there  are no examples or labels at that time.